### ISSUE TITLE
Tags do not propagate to ALB listeners and ALB listener rules v2.1.3

### ISSUE THREAD

**Issue by alexmnyc** [kubernetes-sigs/aws-load-balancer-controller#1966]
```
aws elbv2 describe-tags --resource-arns arn:aws:elasticloadbalancing:us-east-1:XXX:listener/app/k8s-poc-d18b516911/7858123e61a0f436/5250d85534ea8239 
{
    "TagDescriptions": [
        {
            "ResourceArn": "arn:aws:elasticloadbalancing:us-east-1:XXX:listener/app/k8s-podpoc-d18b516911/7858123e61a0f436/5250d85534ea8239",
            "Tags": []
        }
    ]
}

aws elbv2 describe-tags --resource-arns arn:aws:elasticloadbalancing:us-east-1:XXX:loadbalancer/app/k8s-poc-d18b516911/7858123e61a0f436
{
    "TagDescriptions": [
        {
            "ResourceArn": "arn:aws:elasticloadbalancing:us-east-1:889283979366:loadbalancer/app/k8s-podpoc-d18b516911/7858123e61a0f436",
            "Tags": [
                {
                    "Key": "ingress.k8s.aws/stack",
                    "Value": "pod-poc"
                },            
                {
                    "Key": "Environment",
                    "Value": "dev"
                },
...

aws elbv2 describe-tags --resource-arns arn:aws:elasticloadbalancing:us-east-1:XXX:targetgroup/k8s-pod-service2-67d7d47db2/88fba93add4e7e1b 
{
    "TagDescriptions": [
        {
            "ResourceArn": "arn:aws:elasticloadbalancing:us-east-1:XXX:targetgroup/k8s-pod-service2-67d7d47db2/88fba93add4e7e1b",
            "Tags": [
                {
                    "Key": "ingress.k8s.aws/stack",
                    "Value": "pod-poc"
                },
                {
                    "Key": "Environment",
                    "Value": "dev"
                },


aws elbv2 describe-tags --resource-arns arn:aws:elasticloadbalancing:us-east-1:XXX:listener-rule/app/k8s-podpoc-d18b516911/7858123e61a0f436/5250d85534ea8239/3b3723c7a152c716 
{
    "TagDescriptions": [
        {
            "ResourceArn": "arn:aws:elasticloadbalancing:us-east-1:XXX:listener-rule/app/k8s-podpoc-d18b516911/7858123e61a0f436/5250d85534ea8239/3b3723c7a152c716",
            "Tags": []
        }
    ]
}

```

**Comment by M00nF1sh** [comment]
Hi, tagging for listener and listener rules will be supported in v2.2.0

**Comment by alexmnyc** [comment]
Great. Thanks

**Comment by kishorj** [comment]
Support is available in the v2.2.0 release.
